src/Security/UserChecker.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace AppBundle\Security;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Exception\AccountNotValidatedException;
+use Symfony\Component\Security\Core\Exception\DisabledException;
+use Symfony\Component\Security\Core\User\UserCheckerInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+class UserChecker implements UserCheckerInterface
+{
+    /**
```

Comment:
Let's remove inheritdoc code block no ? 
cc @atamano @illusionOfParadise @Remg @thomasbisignani

Reply:
Last month @HeahDude said me that it helps to understand it's implemented method. :s